config.js

```diff
@@ -1134,6 +1136,16 @@ function validate_nc_master_keys_config(config_data) {
     }
 }
 
+function set_nc_nsfs_config_to_env(config_obj) {
```

Comment:
please remove the nsfs from the funsion name, we use nc as the name of the deployment type and were asked to remove `nsfs` usage in general on nc related files/functions/vars etc
in NSFS_ENABLE_DYNAMIC_SUPPLEMENTAL_GROUPS it's ok to use because it's a feature used by both deployment types